packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -1,6 +1,8 @@
 import merge from 'deepmerge'
 import logger from '@wdio/logger'
+import path from 'path'
```

Comment:
Let's import node modules before npm packages. I realise now we might want to eslint this.